tests/unit/test_cf_config.py

```diff
@@ -103,27 +89,12 @@ def test_instantiation_invalid_configuration_file_format(self):
                 config_file='tests/unit/data/ATL03_example.dmr',
             )
 
-    def test_get_cf_attributes_all(self):
```

Comment:
This test has been deprecated, because of the change in behaviour for `CFConfig.get_cf_attributes` (there is no longer a branch that returns all CF Overrides for a `CFConfig` object. I'll talk more about that in place...